src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -210,22 +205,28 @@ private Command prepareGet(String arguments, ItemContainer list) {
      *
      * @param arguments The additional arguments after command word.
      * @return A string indicating parse success or failure.
+     * @throws IllegalFormatException when the input format is wrong
      */
-    private String prepareEdit(String arguments) {
+    private Command prepareEdit(String arguments, ItemContainer list) throws IllegalFormatException {
         final Matcher matcher = EDIT_ITEM_DATA_ARGS_FORMAT.matcher(arguments.trim());
         // Validate arg string format
         if (!matcher.matches()) {
-            return String.format(
-                    CORRECT_COMMAND_MESSAGE_STRING_FORMAT, EDIT_ITEM_DATA_ARGS_FORMAT_STRING);
+            throw new seedu.duke.model.exception.IllegalFormatException(String.format(
```

Comment:
Don't need "seedu.duke.model.exception."?

Reply:
oh ya I have corrected it and update in PR